### PR TITLE
Update to use wikit: 2.0.0-alpha.1 

### PR DESCRIPTION
**Pull request by Ladsgroup**
- Takes advantage of Dropdown menu
 - Set tag for property lookup search results

### CODE REVIEW

File: src/components/PropertyLookup.vue [hunk 1]
```diff
@@ -34,7 +36,13 @@ export default Vue.extend( {
 	},
 	watch: {
 		async search( newSearchString: string ): Promise<void> {
-			this.searchResults = await this.$store.dispatch( 'searchProperties', newSearchString );
+			const searchResults = await this.$store.dispatch( 'searchProperties', newSearchString );
+			this.searchResults = searchResults.map( ( key: MenuItem & SearchResult ) => {
```
Comment (micgro42) [inline src/components/PropertyLookup.vue]: Can we move this into the action?

File: src/components/ValueTypeDropDown.vue [hunk 2]
```diff
@@ -57,12 +72,7 @@ export default Vue.extend( {
 			width: 256px;
 			height: 32px;
 			margin-inline-start: $dimension-layout-xsmall;
-			margin-block-start: 23px;
-			border-color: #a2a9b1;
-			border-style: solid;
-			border-width: 1px;
-			border-radius: 2px;
-			appearance: auto;
+			margin-block-start: 21px;
```
Comment (bereket-WMDE) [inline src/components/ValueTypeDropDown.vue]: shouldn't all this style be removed ? 
AFAIK, the styles are already implemented in the DS. 
just like in the lookup component

File: src/components/ValueTypeDropDown.vue [hunk 3]
```diff
@@ -4,20 +4,12 @@
 			class="querybuilder__dropdown-label">
 			Value Type
 		</label>
-		<select
+		<Dropdown
 			class="querybuilder__dropdown-select"
-			v-model="selected"
+			v-model="selectedItem"
```
Comment (micgro42) [inline src/components/ValueTypeDropDown.vue]: No, we can make all that much simpler now. We basically just listen to the `input` event and re-emit a cleaned version that contains only the value we care about. Conversely, we can set the `value` prop here directly.

Then we can also drop all the `watch` methods and the entire local state. We would then only have something like two computed methods and an event listener left

File: src/components/ValueTypeDropDown.vue [hunk 4]
```diff
@@ -26,12 +18,19 @@
...
 	name: 'ValueTypeDropDown',
 	data() {
 		return {
-			selected: PropertyValueRelation.Matching,
-			optionItems: PropertyValueRelation,
+			selectedItem: {
+				label: PropertyValueRelation.Matching,
+				description: '',
+			},
+			optionItems: Object.values( PropertyValueRelation ).map( ( value: PropertyValueRelation ) => {
```
Comment (micgro42) [inline src/components/ValueTypeDropDown.vue]: Those option items should have translations. They could have the form of
```
{ 
  value: PropertyValueRelation.Matching,
  label: i18n('query-builder-property-lookup-' + PropertyValueRelation.Matching ),
}
```
 or something like that

**Review by micgro42 — CHANGES_REQUESTED** [review]

File: src/components/ValueTypeDropDown.vue (hunk 2 above)
Comment (Ladsgroup) [reply src/components/ValueTypeDropDown.vue]: I removed most of them, but If I remove the rest, it'll look like this:
image

File: src/components/PropertyLookup.vue (hunk 1 above)
Comment (Ladsgroup) [reply src/components/PropertyLookup.vue]: It needs i18n and that's not accessible in outside of the vue object. I can use it but it'll be pretty messy.

File: src/components/ValueTypeDropDown.vue (hunk 3 above)
Comment (Ladsgroup) [reply src/components/ValueTypeDropDown.vue]: I wrestled with it for hours before making this PR, I admit I don't know vue well enough to do what you mean and make it work. Can you try it?

File: src/components/ValueTypeDropDown.vue (hunk 2 above)
Comment (bereket-WMDE) [reply src/components/ValueTypeDropDown.vue]: hmm... @micgro42 could you verify that we don't have any style defined for the lookup component in the QB ?
if so, should it have been the same case with the dropdown ?
thanks

File: src/components/ValueTypeDropDown.vue (hunk 4 above)
Comment (Ladsgroup) [reply src/components/ValueTypeDropDown.vue]: I noticed the problem way before the upgrade, I feel this is out of scope of this PR, we should fix it soon but separately

File: src/components/PropertyLookup.vue (hunk 1 above)
Comment (micgro42) [reply src/components/PropertyLookup.vue]: Mh, I'll have a look. Having the i18n here, but the assigning the tag in the action should work and already make things a lot nicer

File: src/components/ValueTypeDropDown.vue (hunk 3 above)
Comment (micgro42) [reply src/components/ValueTypeDropDown.vue]: Will do!

File: src/components/ValueTypeDropDown.vue (hunk 4 above)
Comment (micgro42) [reply src/components/ValueTypeDropDown.vue]: I think that is fair. Could you create a ticket in the current iteration board for this while I look at the other issues?

File: src/components/ValueTypeDropDown.vue (hunk 2 above)
Comment (micgro42) [reply src/components/ValueTypeDropDown.vue]: will have a look

Comment (micgro42) [reply src/components/ValueTypeDropDown.vue]: I moved the responsibility of arranging the child components to the QueryCondition component

**Review by micgro42 — APPROVED** [review]

**Pull request merged**